lib/service.js

```diff
@@ -12,6 +17,13 @@ class Service {
     this.host = host || process.env.BUZZI_HOST || 'https://core.buzzi.io';
     this.id = id || process.env.BUZZI_API_ID;
     this.secret = secret || process.env.BUZZI_API_SECRET;
+    const validator = new ServiceValidator();
+    const proxy = new Proxy(this, {
+      get(target, key) {
+        return validator[key] ? validator[key](target, key) : target[key];
+      }
+    });
+    return proxy;
```

Comment:
@tttmaximttt - I would remove the Proxy validator functionality for now and keep it simple.  Just put the validation and parameter processing in the same "uploadFiles" function.